src/main/java/com/cleanroommc/groovyscript/compat/mods/advancedrocketry/AdvancedRocketry.java

```diff
@@ -0,0 +1,19 @@
+package com.cleanroommc.groovyscript.compat.mods.advancedrocketry;
+
+import com.cleanroommc.groovyscript.compat.mods.GroovyPropertyContainer;
+
+public class AdvancedRocketry extends GroovyPropertyContainer {
+
+    public final ElectricArcFurnace arcFurnace = new ElectricArcFurnace();
+    public final PrecisionAssembler assembler = new PrecisionAssembler();
+    public final Crystallizer crystallizer = new Crystallizer();
+    public final CuttingMachine cutting = new CuttingMachine();
+    public final Electrolyser electrolyser = new Electrolyser();
+    public final Lathe lathe = new Lathe();
+    public final ChemicalReactor reactor = new ChemicalReactor();
+    public final RollingMachine rolling = new RollingMachine();
+    public final Centrifuge centrifuge = new Centrifuge();
+    public final PrecisionLaserEtcher laserEtcher = new PrecisionLaserEtcher();
+    // No plate presser integration because in-world crafting already handles it. Use JSON recipes to remove those recipes.

```

Comment:
> it just isnt appearing in the dev environment????

Ah I know, I think it's because someone removed the `ore("ingotIron")` from iron ingot. I'm like 99% sure it's that.

Reply:
banger. we should probably disable the `postInit/vanilla.groovy` script via adding `//no_run` on the first line.

Reply:
or add `commented=true` to the entries that do destructive operations (like removing iron from the game)